tests/pipeline/test_us_equity_pricing_loader.py

```diff
@@ -175,43 +175,43 @@
     [
         # Before query range, should be excluded.
         {'declared_date': Timestamp('2015-05-01', tz='UTC').to_datetime64(),
-         'ex_date': Timestamp('2015-06-01', tz='UTC').to_datetime64(),
+         'ex_date': Timestamp('2015-06-02', tz='UTC').to_datetime64(),
```

Comment:
Because the patch makes it so that the previous date (as our current internal calculations do) is used as the effective date.
The test data needs to reflect the change by moving up the `ex_date` so that the original `effective_dates` are used.

That being said, we should check that the date we are using for the dividend supports that behavior.

This assumes that the date of the `ex_date` would be the midnight before the close with same date.
e.g. a dividend with an `ex_date` of Feb. 3rd would be dispersed between Feb. 2nd and Feb 3rd.


Tried to call this out with the line in the message: `Also, use prev close as the effective date`